### PR TITLE
docs: specify to use yarn v1 not v2

### PR DESCRIPTION
**Pull request by JakubKoralewski**
got some errors with yarn v2, so had to downgrade
to even start working, I think it's a good idea
to put it in the README

### CODE REVIEW

**Comment by dwelle** [comment]
Actually, it seems to be a yarn bug that's been fixed: https://github.com/yarnpkg/berry/issues/2935

File: README.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ These instructions will get you a copy of the project up and running on your loc
 #### Requirements
 
 - [Node.js](https://nodejs.org/en/)
-- [Yarn](https://yarnpkg.com/getting-started/install)
+- [Yarn](https://yarnpkg.com/getting-started/install) (v1)
```
Comment (dwelle) [inline README.md:96]:
```suggestion
- [Yarn](https://yarnpkg.com/getting-started/install) (v1, or [v2.4.1+](https://github.com/yarnpkg/berry/blob/master/CHANGELOG.md#241))
```

**Review by dwelle — APPROVED** [review]

**Comment by dwelle** [comment]
@JakubKoralewski lemme know if you were indeed on yarn <2.4.1 or this is a different bug.

**Comment by JakubKoralewski** [comment]
By downgrading I lost the ability to tell which v2 exactly I was on, but now I `yarn set version berry` again which put me on `2.4.2` and the issue is gone.

File: README.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ These instructions will get you a copy of the project up and running on your loc
 #### Requirements
 
 - [Node.js](https://nodejs.org/en/)
-- [Yarn](https://yarnpkg.com/getting-started/install)
+- [Yarn](https://yarnpkg.com/getting-started/install) (v1 or [v2.4.1+](https://github.com/yarnpkg/berry/blob/master/CHANGELOG.md#241))
```
Comment (dwelle) [inline README.md:96]:
```suggestion
- [Yarn](https://yarnpkg.com/getting-started/install) (v1 or v2.4.2+)
```

**Comment by dwelle** [comment]
Cool. I've changed the readme to `v.2.4.2+` because it's unclear whether `2.4.1` works. The issue mentions 2.4.2, but it wasn't in changelog.

**Comment by dwelle** [comment]
Thanks!

**Pull request merged**